src/analysis/mean_inequalities.lean

```diff
@@ -12,8 +12,9 @@ import measure_theory.integration
 /-!
```

Comment:
You still import `measure_theory.integration` at the beginning of the file, is this an oversight?

Reply:
Yes, definitely an oversight, thanks!